CHANGELOG.md

```diff
@@ -24,6 +24,28 @@ CHANGELOG
 Swift 5.0
 ---------
 
+* [SE-0235][]:
+
+  The standard library now contains a `Result<Success, Failure: Error>` type to 
+  manually model potentially error-producing code.
+  
+  ```swift
+  enum Result<Success, Failure: Error> {
+      case success(Success)
+      case failure(Failure)
+  }
+  ```
+  
+  This type serves a separate but complementary role to Swift's existing and 
+  automatic error handling using `try` / `catch`. In addition to it's usefulness 
```

Comment:
```suggestion
```

Describing how they differ can be concentrated into the following sentence; there does not need to be a phrase to introduce well known language keywords.